internal/rundata/dut.go

```diff
@@ -120,30 +125,47 @@ func (di *dutInfo) setFromComponentOS(ctx context.Context, y components.Y) {
 	}
 }
 
-// setFromLLDP sets dutInfo vendor from LLDP system-description (Juniper).  This is less
-// reliable because LLDP config can be changed.
+// setFromLLDP sets dutInfo vendor (Juniper, older Nokia) and osver (older Nokia) from
```

Comment:
Maybe we don't need to support the `older Nokia`, as the needed feature will be available in next build soon?